props/saber_BC_buttons.h

```diff
@@ -2528,16 +2531,19 @@ any # of buttons
 #endif
 
 // Force
-      case EVENTID(BUTTON_POWER, EVENT_SECOND_CLICK_LONG, MODE_ON):
-        if ((spam_blast_ || on_pending_) || (overlap_timer_initialized_ && !overlap_delay_timer_.isTimerExpired())) return false;
-        SaberBase::DoForce();
-        overlap_timer_initialized_ = true;
-        return true;
-
 // Toggle Battle Mode
-      case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON | BUTTON_POWER):
-        ToggleBattleMode();
+      case EVENTID(BUTTON_POWER, EVENT_SECOND_CLICK_LONG, MODE_ON):
+#if (NUM_BUTTONS != 1) || !defined(BC_DUAL_BLADES)  // 1 btn with dual blades has different control
```

Comment:
This #if will leave the "case", but it will fall through to the next thing.
If that's what you actually wanted, add a comment that explains it.
If not, move the #if up one line....

Reply:
oops. thanks for the catch. Fixed.